enterprise/server/cmd/server/BUILD

```diff
@@ -87,6 +87,9 @@ go_binary(
 container_image(
     name = "base_image",
     base = "@buildbuddy_go_image_base//image",
+    files = [
+        "@aws_rds_certs//file",
+    ],
```

Comment:
Can we move this to the //static target so it's also included in the binary bundle and will work for non-docker folks?